yangvalidator/v1/views.py

```diff
@@ -90,7 +96,7 @@ def index(request):
 
 
 def create_output(url, for_datatracker=False):
-    workdir = mkdtemp()
+    workdir = mkdtemp()  # noqa: F405
```

Comment:
Are these `# noqa: F405` needed after fixing the import?

Reply:
thanks, removed them